full-service/src/service/ledger.rs

```diff
@@ -176,8 +176,7 @@ where
     fn get_txo_object(&self, txo_id_hex: &str) -> Result<TxOut, LedgerServiceError> {
         let conn = self.get_conn()?;
         let txo_details = Txo::get(txo_id_hex, &conn)?;
-
-        let txo: TxOut = mc_util_serial::decode(&txo_details.txo)?;
+        let txo = self.ledger_db.get_tx_out_by_index(self.ledger_db.get_tx_out_index_by_public_key(&txo_details.public_key()?)?)?;
```

Comment:
For the sake of readability, I would recommend breaking this up into multiple lines.

In fact, this should probably be refactored into the ledger.rs file as a helper function, since it seems duplicated a few times